tests/test_pipelines_common.py

```diff
@@ -9,4 +18,79 @@ class PipelineTesterMixin:
     equivalence of dict and tuple outputs, etc.
     """
 
-    pass
+    def test_save_load(self):
```

Comment:
nice test! 

IMO we could add a couple of more tests:

- convert to float16 save and load again and check that it's fp16
- retrieve all optional components (see https://github.com/huggingface/diffusers/blob/daebee0963d2b39fb3fa9532ab271a91674c4070/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L70), set them all to None, save & load and check a) that pipeline works